examples/train_unconditional.py

```diff
@@ -171,15 +181,26 @@ def transforms(examples):
 
                 # denormalize the images and save to tensorboard
                 images_processed = (images * 255).round().astype("uint8")
+                epoch06 = str(epoch).zfill( 6 )
+                if args.timestamp_test_samples:
+                    test_samples = datetime.now().strftime(f"test_samples-%Y-%m-%d+%H-%M-%S+{epoch06}")
```

Comment:
This looks like a bit of leftover debugging code (the `06` part in particular`)?

Reply:
The 06 bit is to keep the test_samples names so that they can be sorted the same either alphabetically or numerically. It should be under the if statement for timestamp_test_samples however.